test/images/scripts/fedora-24.setup

```diff
@@ -98,6 +102,9 @@ docker build -t cockpit/base /var/tmp/cockpit-base
 # docker images that we need for integration testing
 /var/lib/testvm/docker-images.setup
 
+# allow nested virtualization
+rmmod kvm_intel && modprobe kvm_intel nested=1
```

Comment:
Unfortunately, I don't have access to an AMD machine now, so I'm relying on documentation.